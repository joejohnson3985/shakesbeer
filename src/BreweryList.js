import React from 'react';
import Breweries from './Breweries.js'

function BreweryList(props) {
  return (
    <div>
      {
        props.filteredBreweries.map(brewery => {
          return (
            <Breweries name={brewery.name}
                       dataset={props.dataset}
                       starredBreweries={props.starredBreweries}
                       updateStarredList={props.updateStarredList}
                       key={Date.now()}
            />
          )
        })
      }
    </div>
  )
}

export default BreweryList;