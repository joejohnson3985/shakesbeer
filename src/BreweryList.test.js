import React from 'react';
import BreweryList from './BreweryList.js';
import { shallow } from 'enzyme';

const mockData = [1,2,3,4]

describe('BreweryList', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow (
            <BreweryList filteredBreweries={mockData} 
                         dataset={mockData}
            />
        )
    });

    it('Should match the snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    });
});