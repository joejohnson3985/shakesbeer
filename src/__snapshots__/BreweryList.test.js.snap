// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BreweryList Should match the snapshot 1`] = `
<div>
  <Breweries
    dataset={
      Array [
        1,
        2,
        3,
        4,
      ]
    }
    key="0"
  />
  <Breweries
    dataset={
      Array [
        1,
        2,
        3,
        4,
      ]
    }
    key="1"
  />
  <Breweries
    dataset={
      Array [
        1,
        2,
        3,
        4,
      ]
    }
    key="2"
  />
  <Breweries
    dataset={
      Array [
        1,
        2,
        3,
        4,
      ]
    }
    key="3"
  />
</div>
`;
